RClasses-common/src/main/java/de/florianmichael/rclasses/common/StringUtils.java

```diff
@@ -142,4 +142,19 @@ public static String smallestOf(final String... strings) {
         }
         return smallestString;
     }
+
+    /**
+     * Normalizes the given enum name to a readable string (e.g. "TEST_ENUM" -> "Test enum"). This method is used to
+     * generate a readable name for the enum. "$" will be removed, "_" will be replaced with " ",
+     * and the first letter will be uppercase.
+     *
+     * @param name The name to normalize
+     * @return The normalized name
+     */
+    public static String normalizeEnumName(String name) {
+        name = name.replace("_", " ");
```

Comment:
Please add a check in case the name string is too short.

Reply:
Done